documentation/source/newsfragments/1309.feature.rst

```diff
@@ -0,0 +1,4 @@
+The colored output can now be disabled by the :envvar:`NO_COLOR` environment variable.
+
+If NO_COLOR is set (regardless of its value) color in the output will be suppressed.
+If COCOTB_ANSI_OUTPUT=1 is set, NO_COLOR will be ignored.
```

Comment:
Now that github supports multiline suggestions, what I meant by [this comment](https://github.com/cocotb/cocotb/pull/1438/files#r382908229) is:
```suggestion
```